stdeb/pypi_simple.py

```diff
@@ -0,0 +1,46 @@
+import re
+import requests
+import stdeb
+
+USER_AGENT = "pypi-install/%s ( https://github.com/astraw/stdeb )" % \
+    stdeb.__version__
```

Comment:
Since the only current use case passes in an explicit `USER_AGENT` anyway, it might be good to remove the duplicated string here and make the default behavior not specify a user agent at all.

Reply:
I forgot to make this change but I agree with it.

Reply:
In 0787a84 I ended up importing the USER_AGENT constant from the downloader. I think adding a download method to the pypi client would also be another way to push this down. But this cuts the duplication although it keeps the default user agent.